src/styles/main.scss

```diff
@@ -262,4 +262,17 @@ formly-array-type {
   content: "" !important;
 }
 
+.mdc-text-field--disabled, 
+.mdc-text-field--disabled .mat-mdc-input-element {
+  color: black !important;
+}
+
+.mdc-text-field--disabled .mdc-floating-label {
+  color: rgba(0, 0, 0, 0.6)
+}
```

Comment:
What's the purpose of this? @CodeKrakken

Reply:
@lukestanley For consistency of style with the labels of regular input fields - they are not black but slightly grey.